### PR TITLE
second commit to begin JS App build

### PR DESCRIPTION
**Pull request by walshsean**
Set Global VAR of Beer and various elements to be used in the page for functionality: forms for the user, buttons, and a container for results.

### CODE REVIEW

File: application.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+	var elements = {
+		container: document.querySelector(".beer-background"),
+		resultsContainer: document.querySelector(".beer-results"),
+		button: document.getElementById("beer-search-button"),
+	  form: document.querySelector(".beer-form"),
+	  input: document.querySelectorAll('input'),
+	  closeOutButton: document.querySelector(".beer-modal-close")
+	};
+
```
Comment (lizzzp1) [inline application.js]: why is spacing all weird? 😞

Comment (lizzzp1) [reply application.js]: There's a mix of single quotes and double

**Comment by walshsean** [comment]
ah sweet! I'll fix! :)

**Review by lizzzp1 — APPROVED** [review]
Looks good otherwise!

**Pull request merged**